tachyon/init.sh

```diff
@@ -7,8 +7,6 @@ if [ -d "tachyon" ]; then
   return 0
 fi
 
-TACHYON_VERSION=0.5.0
-
 # Github tag:
 if [[ "$TACHYON_VERSION" == *\|* ]]
 then
```

Comment:
Could you add a print message here something like: `echo "Tachyon git hashes are not yet supported. Please specify a Tachyon release version."`